src/entities/project.ts

```diff
@@ -299,6 +299,12 @@ class Project extends BaseEntity {
   @Field(type => ProjectUpdate, { nullable: true })
   projectUpdate?: any;
 
+  @Field(type => [ProjectUpdate], { nullable: true })
+  projectUpdates?: ProjectUpdate[];
+
+  @Field(type => String, { nullable: true })
+  adminBroBaseUrl: string;
```

Comment:
I found no other way to pass the env var to the react view. So I added a virtual attribute I fill with the env var, and pass it as props.